furnitureStoreDBPopulate.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+//RUN ONCE AFTER DB (furniture_store) STRUCTURE HAS BEEN BUILT
+require('vendor/autoload.php');
+
+$jsonRequest = file_get_contents('https://dev.io-academy.uk/resources/furniture/furniture.json');
```

Comment:
To make your code more self-contained, it would be better to download the json file and host it locally: that way the database could be recreated even if our servers go down!